src/ApiPort/ApiPort.Vsix/ApiPort.Vsix.csproj

```diff
@@ -11,7 +11,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>ApiPort.Vsix</RootNamespace>
     <AssemblyName>ApiPort.Vsix</AssemblyName>
-    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
```

Comment:
Again, verify what the minimum support for VS 2017 is

Reply:
minimum support is 4.6.1 as stated above.